package.json

```diff
@@ -1,99 +1,126 @@
+		"scroll",
+		"unichain",
+		"ten",
+		"worldchain"
+	],
+	"bin": {
+		"witeth": "src/bin/index.cjs",
+		"witnet-ethers": "src/bin/index.cjs",
+		"witnet-solidity": "src/bin/index.cjs",
+		"randomizer": "src/bin/bots/randomizer.cjs"
+	},
+	"exports": {
+		".": {
+			"default": "./dist/src/index.js",
+			"require": "./dist/src/index.js",
+			"types": "./dist/src/*.d.ts"
+		},
+		"./contracts/*": "./contracts/",
+		"./utils": {
+			"default": "./dist/src/lib/utils.js",
+			"require": "./dist/src/lib/utils.js",
+			"types": "./dist/src/lib/*.d.ts"
+		}
+	},
+	"files": [
+		"artifacts",
+		"build/contracts",
+		"dist",
+		"contracts",
+		"src/bin"
+	],
+	"scripts": {
+		"build": "tsc -p tsconfig.json",
+		"clean": "npm run clean:artifacts && npm run clean:build && npm run clean:cache && npm run clean:dist",
+		"clean:artifacts": "node ./scripts/clean.cjs artifacts",
+		"clean:build": "node ./scripts/clean.cjs build",
+		"clean:cache": "node ./scripts/clean.cjs cache",
+		"clean:dist": "node ./scripts/clean.cjs dist",
+		"compile": "npx truffle compile --all --config ./truffle-config.cjs",
+		"console": "npx truffle console",
+		"fmt:js": "npx biome check",
+		"fmt:sol": "solhint --max-warnings 0 \"contracts/**/*.sol\"",
+		"fmt!:js": "npx biome check --write --unsafe",
+		"fmt!:sol": "solhint --max-warnings 0 \"contracts/**/*.sol\" --fix",
+		"fmt!": "pnpm run fmt!:js && pnpm run fmt!:sol",
+		"fmt": "pnpm run fmt:js && pnpm run fmt:sol",
+		"deploy": "npx truffle migrate --config ./truffle-config.cjs",
+		"postinstall": "node ./scripts/postinstall.cjs",
+		"prepare": "npx truffle compile --all --config ./truffle-config.cjs && npx hardhat compile --force && npm run build && node ./scripts/prepare.cjs",
+		"verify": "npx hardhat run ./scripts/verify.js 2>&1",
+		"witeth": "cross-env-shell WITNET_SDK_RADON_ASSETS_PATH='../..' node ./src/bin/index.cjs"
+	},
+	"dependencies": {
+		"@witnet/sdk": "../witnet-toolkit",
+		"axios": "^1.12.2",
+		"cbor": "^10.0.11",
+		"cross-env": "^10.1.0",
+		"dotenv": "^17.2.3",
+		"ethers": "^6.15.0",
+		"ethrpc-gateway": "^2.3.1",
+		"inquirer": "^8.2.7",
+		"json-bigint": "^1.0.0",
+		"lodash.merge": "^4.6.2",
+		"moment": "^2.30.1",
+		"node-cron": "^4.2.1",
+		"promise-poller": "^1.9.1"
+	},
+	"devDependencies": {
+		"@biomejs/biome": "^2.2.6",
+		"@nomicfoundation/hardhat-verify": "^3.0.2",
+		"@openzeppelin/contracts": "^5.4.0",
+		"@openzeppelin/contracts-upgradeable": "^5.4.0",
+		"@types/json-bigint": "^1.0.4",
+		"@types/lodash.merge": "^4.6.9",
+		"@types/node": "^24.9.1",
+		"ado-contracts": "1.0.0",
+		"chai": "^6.2.0",
+		"custom-error-test-helper": "^1.0.6",
+		"eth-gas-reporter": "^0.2.27",
+		"eth-helpers": "^1.3.1",
+		"hardhat": "^3.0.6",
+		"nanoassert": "^3.0.1",
+		"sha3-wasm": "^1.0.0",
+		"solhint": "^6.0.1",
+		"truffle": "^5.11.5",
+		"truffle-assertions": "^0.9.2",
+		"truffle-flattener": "^1.6.0",
+		"truffle-plugin-verify": "^0.6.7",
+		"typescript": "^5.9.3",
+		"utf-8-validate": "^5.0.2",
```

Comment:
Are we using the following dependencies?

- [ ] utf-8-validate
- [ ] truffle-flattener
- [ ] truffle-assertions
- [ ] custom-error-test-helper
- [ ] @types/json-bigint
- [ ] json-bigint
- [ ] axios